portugues/colabora/comdiferencas/diff/corrigeepullrequest/corrigeepullrequest.md

```diff
@@ -0,0 +1 @@
+O responsável pelo repositório de origem verifica que outros novos 13 bugs apareceram depois das suas alterações e envia uma notificação para você, você então:
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.